include/swift/Parse/SyntaxParsingCache.h

```diff
@@ -86,11 +93,12 @@ class SyntaxParsingCache {
   getReusedRegions(const SourceFileSyntax &SyntaxTree) const;
 
   /// Translates a post-edit position to a pre-edit position by undoing the
-  /// specified edits.
+  /// specified edits. Returns \c true if pre-edit position cannot be
+  /// determined (for example, the position is in a region which is inserted by
+  /// a edit).
```

Comment:
Nitpick: **an** edit

Reply:
While looking through the PR, it somehow took me fairly long to understand when `false` is returned. I think it might help, if you add something along the lines of `returns \c true if no pre-edit position exists because the post-edit position has been inserted by an en edit`. It's up to you to decide though. I understand it now.